docs/source/json_parser.py

```diff
@@ -0,0 +1,33 @@
+import os
+import json
+
+ROOT_DIR = os.getcwd()
+SOURCE_DIR = os.path.join(ROOT_DIR, "docs/source/")
+OUT_FILEPATH = os.path.join(SOURCE_DIR, "parsed_info.json")
+info_json_file_path = os.path.join(SOURCE_DIR, "_tmp/", "info.json")
+
+with open(info_json_file_path) as f:
+    info_json_raw = json.load(f)
+
+def dist_str_to_dict(dist: str) -> dict:
+        pkg = dist.rsplit('-', 2)
+        pkg_split = pkg[2].split('_')
+
+        pkg_name = pkg[0]
+        pkg_version = pkg[1]
+        pkg_hash = pkg_split[0]
+        pkg_build_num = pkg_split[1].split('.')[0]
```

Comment:
```suggestion
        pkg_build_num = int(pkg_split[1][:-6])
```

We know that it ends with `.conda` and we also know that this is always an integer.

Reply:
Thank you!